src/test/java/org/opensearch/neuralsearch/common/BaseNeuralSearchIT.java

```diff
@@ -62,6 +62,12 @@ public abstract class BaseNeuralSearchIT extends OpenSearchSecureRestTestCase {
 
     @Before
     public void setupSettings() {
+        if (isUpdateClusterSettings()) {
+            updateClusterSettings();
+        }
+    }
+
+    protected void updateClusterSettings() {
```

Comment:
`@VisibleForTesting`?

Reply:
I think that would be redundant, it's base class for tests